halyard-deploy/src/main/resources/services/vault/client/startup/mount-config.py

```diff
@@ -5,31 +5,37 @@
 import json
 import logging
 import subprocess
-import sys
 import os
 import grp
 import pwd
+import time
 
 def mkdirs(path):
     subprocess.check_call(["mkdir", "-p", path])
 
+
+def subprocess_retry(supplier, retries):
+    if retries < 0:
+        logging.fatal("All retries attempted")
+        raise subprocess.CalledProcessError("")
```

Comment:
Seems like you could just join the command list and add it as a param: `' '.join(cmd_tokens)` and then include that in the raised exception.